invenio_admin/ext.py

```diff
@@ -62,12 +62,20 @@ def init_app(self, app, entry_point_group='invenio_admin.views',
         # Load administration interfaces defined by entry points.
         if entry_point_group:
             for ep in pkg_resources.iter_entry_points(group=entry_point_group):
-                modelview, model = ep.load()
+                adminview_dict = ep.load()
+                assert 'model' in adminview_dict, \
+                    "Admin's entrypoint dictionary must define the 'model'"
+                assert 'modelview' in adminview_dict, \
+                    "Admin's entrypoint dictionary must define the 'modelview'"
+                model = adminview_dict['model']
```

Comment:
That will modify the dictionary on the other end, I believe making a copy is safer.